week-2/G-conditionals-2/exercise-3.js

```diff
@@ -9,20 +9,32 @@
 */
 
 function calculateGrade(mark) {
-
+ if (mark > 80){
+   return "A";
+  }
+  else if (mark < 80 && mark > 60){
```

Comment:
Once you hit the else statement, the pairing if (e.g. from line 12) has already been evaluated as false. So you don't actually need to repeat the inverse condition here like `mark < 80`